MODULE.bazel

```diff
@@ -17,4 +17,21 @@ node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
 node.toolchain(name = "nodejs")
 use_repo(node, "nodejs_toolchains")
 
-register_toolchains("@nodejs_toolchains//:all")
+# Toolchain registration under bzlmod should match the order of WORKSPACE registration
```

Comment:
You can leave this one here for the context: https://github.com/bazelbuild/bazel/issues/19645